libr/core/cmd_print.c

```diff
@@ -2575,21 +2580,59 @@ static int cmd_print(void *data, const char *input) {
 					r_list_sort (f->bbs, (RListComparator)bbcmp);
 					if (input[2] == 'j') {
 						r_cons_printf ("[");
+						for (locs_it; locs_it && (tmp_func = locs_it->data); locs_it = locs_it->n) {
+							if (tmp_func->addr < f->addr) {
+								r_list_foreach (f->bbs, iter, b) {
+									r_core_cmdf (core, "pDj %"PFMT64d" @0x%"PFMT64x, b->size, b->addr);
+									if (iter->n) {
+										r_cons_printf (",");
+									}
+								}
+							} else {
```

Comment:
maybe is better to put the smaller block first .. so: if (f->addr < tmp_func->addr) { break; } .without the else block